packages/mui-joy/src/SvgIcon/SvgIcon.test.tsx

```diff
@@ -144,7 +153,10 @@ describe('<SvgIcon />', () => {
       this.skip();
     }
 
-    const { container } = render(<SvgIcon ownerState={{ fontSize: 'sm' }}>{path}</SvgIcon>);
+    const { container } = render(
```

Comment:
this test was specifically added here https://github.com/mui/material-ui/pull/34429/files because of this issue https://github.com/mui/material-ui/issues/34056

Should I delete this test?

Reply:
You are right.. thanks for investigating that and letting me know. The PR looks good enough. Merging it now.